src/fem/dofhandler.hh

```diff
@@ -346,6 +259,13 @@ public:
 	 * @param loc_nb Local index of neighbour.
 	 */
 	inline LongIdx nb_index(int loc_nb) const { return nb_4_loc[loc_nb]; }
+	
+	/**
+     * @brief Return number of dofs on given cell.
+     * 
+     * @param cell Cell accessor.
+     */
+	unsigned int n_dofs(ElementAccessor<3> cell) const;
```

Comment:
What is CellIterator vs. ElementAccessor. Documentation is bit confusing.
Anyway, is it enough to provide iterating over the local cells with a dedicated DoFHandler iterator with automaticaly updated informations, then we can remove both get_loc_dof_indices and n_dofs and can support also getting particular dofs for faces.

... can be part of further development, possibly can be done by David.

Reply:
CellIterator removed and replaced by ElementAccessor<3>.